pages/index.tsx

```diff
@@ -2,13 +2,30 @@ import dynamic from 'next/dynamic';
 import { ThemeProvider } from '~/hooks/useTheme';
 
 import { PLACES } from '~/data/places';
+import Sidebar from '~/components/Sidebar';
+import { menu } from '~/lib/utils';
 
 const Map = dynamic(() => import('~/components/Map'), { ssr: false });
 
 export default function Home() {
   return (
-    <ThemeProvider>
-      <Map pins={PLACES} />
-    </ThemeProvider>
+    <div>
+      <input id="menu_checkbox" type={'checkbox'} onClick={menu} />
+      <label className="menu_label" htmlFor="menu_checkbox" title="Menu">
+        <div></div>
+        <div></div>
+        <div></div>
+      </label>
```

Comment:
All this should be a `Menu` component :muscle:

Reply:
Got it, I'm just not quite sure why having a `isOpen` state would be useful. I would need to set that state manually according to when the Menu checkbox is checked or unchecked right? Not sure I understood the assignment ahaha. I need to learn more about it.